independent-publisher-connectors/Shopify Admin Content/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2896 @@
+                          "type": "integer",
+                          "format": "int32",
+                          "description": "width"
+                        },
+                        "height": {
+                          "type": "integer",
+                          "format": "int32",
+                          "description": "height"
+                        },
+                        "src": {
+                          "type": "string",
+                          "description": "src"
+                        }
+                      },
+                      "description": "image"
+                    }
+                  },
+                  "description": "article"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/blogs.json": {
+      "get": {
+        "description": "Retrieves a list of all blogs for your store",
+        "summary": "Get a list of all blogs",
+        "tags": [
+          "Blog"
+        ],
+        "operationId": "ListBlogs",
+        "deprecated": false,
+        "produces": [
+          "application/json"
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "description": "Content Type",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "List of blogs response",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "blogs": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "id": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "id"
+                      },
+                      "handle": {
+                        "type": "string",
+                        "description": "Handle"
+                      },
+                      "title": {
+                        "type": "string",
+                        "description": "title"
+                      },
+                      "updated_at": {
+                        "type": "string",
+                        "description": "Updated at"
+                      },
+                      "commentable": {
+                        "type": "string",
+                        "description": "commentable"
+                      },
+                      "feedburner": {
+                        "type": "string",
+                        "description": "feedburner"
+                      },
+                      "feedburner_location": {
+                        "type": "string",
+                        "description": "feedburner_location"
```

Comment:
@thatapiguy Can you please provide valid description values for similar properties, like you did for most of the description values.